app/views/comments/create.js.erb

```diff
@@ -1,23 +1,43 @@
+// this code runs after a new comment is posted.
+//   1. it inserts the new comment into the DOM
+//   2. it also creates image upload functionality for the new comment form (see /app/assets/javascripts/editorToolbar.js)
+
+// the new comment will have an 'edit comment' form that needs image upload functionality:
+let fileUploadElements = [
+  {
+    selector: '#comment-form-body-edit-<%= @comment.cid %>',
+    isButton: false
+  },
+  {
+    selector: '#dropzone-small-edit-<%= @comment.cid %>',
+    isButton: true
+  }
+];
+// if new comment is NOT a reply to another comment
 <% if @comment.reply_to.nil? %>
-  $("#comments-list").append('<%= escape_javascript(render :partial => "notes/comment", :locals => {comment: @comment, answer_id: @answer_id}) %>')
+  $("#comments-list").append('<%= escape_javascript(render :partial => "notes/comment", :locals => { comment: @comment }) %>');
+   // the new comment will also have its own reply section
+  // this reply comment form needs image upload functionality
+  fileUploadElements = fileUploadElements.concat([
+    {
+      selector: '#comment-form-body-reply-<%= @comment.cid %>',
+      isButton: false
+    },
+    {
+      selector: '#dropzone-small-reply-<%= @comment.cid %>',
+      isButton: true
+    }
+  ]);
```

Comment:
Reminder that this code is for attaching image upload functionality to fresh comments.

I realized that a fresh comment can either:
1. Be a comment NOT responding to another comment (will have both an edit form and a reply form).
2. Be in response to another comment (will have ONLY an edit form).

Therefore, if a comment is a response, image upload functionality should only be attached to the edit form (the reply form doesn't exist).

That's the main change I'm making in this PR.